platform-bungeecord/src/main/java/net/kyori/adventure/platform/bungeecord/BungeeReflection.java

```diff
@@ -0,0 +1,61 @@
+package net.kyori.adventure.platform.bungeecord;
+
+import java.lang.invoke.MethodHandle;
+import java.lang.invoke.MethodHandles;
+import java.lang.invoke.MethodType;
+
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Reflection utilities for accessing legacy BungeeCord methods
+ */
+public class BungeeReflection {
+
+  private static final MethodHandles.Lookup LOOKUP = MethodHandles.lookup();
```

Comment:
Probably move this below line 17 and probably also change the whitespaes stuff to reflect the reference I provided
Reference: https://github.com/KyoriPowered/adventure-platform/blob/de6e761bad4d6ddf63fdb3d7d5c825c88a0488a8/platform-bukkit/src/main/java/net/kyori/adventure/platform/bukkit/MinecraftReflection.java#L43